tools/docker/esp32/Makefile

```diff
@@ -1,11 +1,20 @@
+e :=
+c := ,
+clist = $(subst $e $e,$c,$(strip $1))
+
 DOCKERFILES = esp32-build
 SPIFFS_CONFIG_PATH = /vfs-fs-spiffs/include/esp32
-TOOLCHAIN_URL = https://github.com/espressif/crosstool-NG/releases/download/esp-2021r2-patch3/xtensa-esp32-elf-gcc8_4_0-esp-2021r2-patch3-linux-amd64.tar.gz
-DOCKER_FLAGS = --build-arg=TOOLCHAIN_URL=$(TOOLCHAIN_URL) \
-               --build-arg=DOCKER_TAG=$(DOCKER_TAG)
+PLATFORMS = arm64 amd64
+
+DOCKER_FLAGS = --build-arg=DOCKER_TAG=$(DOCKER_TAG) \
+               --platform $(call clist, $(foreach i, $(PLATFORMS), linux/$(i)))
 
 include ../docker.mk
 
-docker-pre-build-esp32-build: mgos_fw_meta.py serve_core mklfs mkspiffs mkspiffs8
+docker-pre-build-esp32-build: mgos_fw_meta.py serve_core \
+                              $(foreach i,$(PLATFORMS),mklfs-$(i)) \
```

Comment:
i think we should create an additional target in docker.mk, call it `mkfs-%` that depends on the 3 targets

Reply:
actually I moved all the platform specific stuff into docker.mk. Apart from (optional) PLATFORMS variable there shouldn't be anything to consider